pipelines/components/image_classifier_keras/setup.py

```diff
@@ -0,0 +1,32 @@
+"""Install dependencies. Usage: python setup.py install
```

Comment:
The first comment line seems to be a bit out of place. Is it supposed to be the first line in the file?
And the header block seems to be doubly commented out (by # and also by """)

Reply:
The setup.py layout is a cut and paste of the same in linear_learner. I used it as a template.